en/docs/guides/user-mgt/managing-user-attributes.md

```diff
@@ -0,0 +1,204 @@
+# Managing User Attributes
+
+When using the WSO2 Identity Server (WSO2 IS) for user and role management, it is
+important to understand how to manage the attributes of users within it.
+In WSO2 IS, each user store attribute can be mapped as a
+claim. Therefore, you need to use the claim management functionality
+available in WSO2 IS to properly map your LDAP/AD/JDBC user
+store attributes with the claim URIs defined by WSO2 IS. You
+can also add different claim URIs and manage them using claim
+management.
+
```

Comment:
conceptual info can go to concepts section and we can simply give a link to it here